freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/dns/doc/DnsModelDescription.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.freeipa.api.v1.dns.doc;
+
+public final class DnsModelDescription {
+    public static final String DNS_RECORD = "DNS record name without the domain. eg. 'ipaserver' from 'ipaserver.clouder.site'";
+
+    public static final String DNS_ZONE = "It's the domain. Like if your FQDN is ipaserver.clouder.site, it's 'cloudera.site'. "
+            + "'168.192.in-addr.arpa' for a reverse record like '5.1.168.192.in-addr.arpa'";
+
+    public static final String CNAME_TARGET_FQDN = "The fully qualified domain name of the host the CNAME should point to.";
+
+    public static final String CREATE_REVERSE = "Tries to create a reverse point for the recorc (PTR). Only if reverse zone already exists";
```

Comment:
Typo: `recorc` -> `record`.

Reply:
thanks, I fixed locally. For now I won't push it.